internal/server/models/group.go

```diff
@@ -13,9 +13,7 @@ type Group struct {
 	CreatedBy         uid.ID
 	CreatedByProvider uid.ID
 
-	Identities []Identity `gorm:"many2many:identities_groups"`
```

Comment:
This field was not used, so removing it was as easy as removing this one line. We can always add it back in the future if we need it, but we'll have to add support to `CreateGroup` to actually use the column.

Right now most code seems to use the `Groups` field on `Identity`.